ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -455,9 +454,9 @@ private class JdbcLedgerDao(
                 .foreach(_.execute())
             )
           } else {
-            for (info @ SubmitterInfo(_, _, commandId, _) <- submitterInfo) {
+            for (info <- submitterInfo) {
```

Comment:
Can we reuse the `storeRejection` method in here? It seems to be almost identical

Reply:
No, each defines its own transaction boundaries. I'll refactor this part outside. Please note that this part is not particularly relevant to the PR, I was just doing some cleaning as this came up along the way (the so-called "boy scout" rule).

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8019/commits/4550a0b0f759efa34d96886772ebb0bee4610f4f